docs/security.html

```diff
@@ -34,6 +34,19 @@ <h3><a id="security_overview" href="#security_overview">7.1 Security Overview</a
 
     It's worth noting that security is optional - non-secured clusters are supported, as well as a mix of authenticated, unauthenticated, encrypted and non-encrypted clients.
 
+    <h3><a id="insecure_by_default" href="#insecure_by_default"> &#9888; WARNING: Insecure By Default</a></h3>
+    <p> While security it optional, Kafka/Kafka Connect do not come with any security enabled by default. This means that Kafka/Kafka Connect instances can be read/written/configured by unauthenticated clients.
```

Comment:
It seems like this paragraph should be merged with the previous one.  Also, given that this is a generic security document, we don't need to reference Kafka Connect specifically, right?

Reply:
agreed; updated